code/modules/mob/living/simple_animal/hostile/terror_spiders/empress.dm

```diff
@@ -0,0 +1,268 @@
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: T5 EMPRESS OF TERROR -------------------------
+// --------------------------------------------------------------------------------
+// -------------: ROLE: ruling over planets of uncountable spiders, like Xenomorph Empresses.
+// -------------: AI: none - this is strictly adminspawn-only and intended for RP events, coder testing, and teaching people 'how to queen'
+// -------------: SPECIAL: Lay Eggs ability that allows laying queen-level eggs. Wide-area EMP and light-breaking abilities.
+// -------------: TO FIGHT IT: run away screaming?
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/EldritchAbomination
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/BiggerBad
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/AuthorityEqualsAsskicking
+// -------------: SPRITES FROM: FoS, http://nanotrasen.se/phpBB3/memberlist.php?mode=viewprofile&u=386
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress
+	name = "Empress of Terror"
+	desc = "The unholy offspring of spiders, nightmares, and lovecraft fiction."
+	spider_role_summary = "Adminbus spider"
+	ai_target_method = TS_DAMAGE_SIMPLE
+	altnames = list ("Terror Empress spider")
+	egg_name = "empress spider eggs"
+
+	icon_state = "terror_queen"
+	icon_living = "terror_queen"
+	icon_dead = "terror_queen_dead"
+
+	maxHealth = 1000
+	health = 1000
+
+	melee_damage_lower = 30
+	melee_damage_upper = 60
+
+	move_to_delay = 5
+	ventcrawler = 1
+
+	idle_ventcrawl_chance = 0
+	ai_playercontrol_allowtype = 0
+	ai_type = TS_AI_AGGRESSIVE
+
+	ranged = 1
+	retreat_distance = 5
+	minimum_distance = 5
+	projectilesound = 'sound/weapons/pierce.ogg'
+	projectiletype = /obj/item/projectile/terrorempressspit
+	force_threshold = 18 // same as queen, but a lot more health
+
+	spider_tier = 5
+	spider_opens_doors = 2
+
+	var/shown_guide = 0 // has the empress player been warned of the chaos that can result from the use of their powers?
+	var/phasing = 0
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/Life()
+	if(stat != DEAD)
+		if(ckey && !shown_guide)
+			shown_guide = 1
+			ShowGuide()
```

Comment:
Why would you **EVER** do something like this in Life()? It's a verb, there's no reason to force show it to somebody, and it could probably just be handled in the mob's Login() proc if you did.